RevenueCatUI/CustomerCenter/ViewModels/ManageSubscriptions/ManageSubscriptionsViewModel.swift

```diff
@@ -41,6 +41,8 @@ final class ManageSubscriptionsViewModel: ObservableObject {
     @Published
     var showPurchases: Bool = false
 
+    @Published var showBalances: Bool = false
```

Comment:
Is this being used? Can't find where 🤔

Reply:
It isn't! This is leftover from an earlier approach during development. I've removed it in 039034f3f9b5f670c4a4dcec1e10f6f220ba623c. Good catch!